setup/kubernetes/local-preview.md

```diff
@@ -21,6 +21,11 @@ Before proceeding, please make sure that you have the following installed:
 
 ## Limitations
 
+### Resource requirements
+
+Make sure your host machine meets the [resource allocation minimums](../requirements)
+necessary to run Coder.
+
```

Comment:
@coadler if you have specifics here, that would be great!

Reply:
fwiw, the script sets the resource requests for coder services to 0 and automatically sets cpu/mem overprovisioning high, plus the cluster setup is controlled by us, so I don't think much of this applies to the kind script. it might make more sense to mention the patform experience will directly be tied to system specs, and some performance drop is to be expected since it's all running inside a docker container.

Reply:
If you'd like to include Colin's caveat, maybe it could be worded as:

> Your experience with the local Coder preview is dependent on your system specs, but please note that you can expect slightly degraded performance due to the deployment running completely inside a Docker container.

And maybe the title would be "Resource allocation and performance" or something like that